### DISCUSSION TITLE
Why not use __tests__ folders?

### DISCUSSION THREAD

**Discussion by pmaier983** [react-hook-form/react-hook-form#1746] [General] [answered]
\_\_tests\_\_ folder's would visually declutter the library and could improve navigation (specifically when navigating the code via GitHub). 

It won't have a impact on performance, but it seems to me that when navigating around this repo on GitHub I have to dodge either the actual code or the tests (depending on what I'm looking for). It might be better to separate the two?

**Comment by bluebill1049** [comment, marked as answer]
hey @pmaier983 thanks for the suggestion, I guess both are valid, more of personal preference. 👍
